content/02.introduction.md

```diff
@@ -10,11 +10,16 @@ Historically, some of the most fatal, inoperable brain tumors, such as diffuse m
 
-Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace. 
-Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma [@doi:10.1097/01.COT.0000421359.88140.a8; @url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-everolimus-tuberous-sclerosis-complex-associated-partial-onset-seizures]; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors [@url:https://www.fda.gov/drugs/drug-approvals-and-databases/fda-approves-pembrolizumab-adults-and-children-tmb-h-solid-tumors]; NTRK inhibitors larotrectinib [@url:https://www.fda.gov/drugs/fda-approves-larotrectinib-solid-tumors-ntrk-gene-fusions] and entrectinib [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-entrectinib-ntrk-solid-tumors-and-ros-1-nsclc] for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-selumetinib-neurofibromatosis-type-1-symptomatic-inoperable-plexiform-neurofibromas]. 
-This is, in part, due to pharmaceutical company priorities and/or concerns regarding toxicity leading, making it challenging for researchers to obtain to new therapeutic agents for pediatric clinical trials.
-Critically, as of August 18, 2020, an amendment to the Pediatric Research Equity Act called the "Research to Accelerate Cures and Equity (RACE) for Children Act" mandates that all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer. 
+Just as cooperation is required for the sharing of specimens and data, rigorous analysis of pediatric brain tumor data requires collaboration, as no one individual researcher or research group is best positioned to perform every analysis needed.
+To leverage diverse expertise, we used an open science model, with features regarded to improve reproducibility of computational research such as analytical code review [@doi:10.1093/aje/kwab092; @doi:10.7287/peerj.preprints.3210v1], throughout the life cycle of our project analyzing PBTA data.
+We anticipated that working in the open would enhance the value of our effort to the pediatric brain tumor research community – works that include data and code sharing may have a greater impact [@doi:10.1371/journal.pone.0000308; @doi:10.1371/journal.pcbi.1008867] – and increase efficiency because others would not need to wait until after publication to build on our work.
```

Comment:
I'm suggesting an alternative here, breaking this into two separates sentences to try to avoid the challenges of splicing. 
```suggestion
We anticipated that a model of open collaboration would enhance the value of our effort to the pediatric brain tumor research community. 
Openly sharing data and code in real time allows others to quickly build upon our work rapidly, and publications that include data and code sharing may have a greater impact [@doi:10.1371/journal.pone.0000308; @doi:10.1371/journal.pcbi.1008867].
```